Setup/InstallData.php

```diff
@@ -1,34 +1,25 @@
 <?php
-/**
- * StoreFront Bazaarvoice Extension for Magento
- *
- * PHP Version 5
- *
- * LICENSE: This source file is subject to commercial source code license
- * of StoreFront Consulting, Inc.
- *
- * @category  SFC
- * @package   Bazaarvoice_Ext
- * @author    Dennis Rogers <dennis@storefrontconsulting.com>
- * @copyright 2016 StoreFront Consulting, Inc
- * @license   http://www.storefrontconsulting.com/media/downloads/ExtensionLicense.pdf StoreFront Consulting Commercial License
- * @link      http://www.StoreFrontConsulting.com/bazaarvoice-extension/
- */
+declare(strict_types=1);
 
 namespace Bazaarvoice\Connector\Setup;
 
-use Magento\Framework\App\State;
-use Magento\Framework\Setup;
-use Magento\Framework\ObjectManagerInterface;
 use Bazaarvoice\Connector;
-use Magento\Sales\Setup\SalesSetup;
-use Magento\Sales\Setup\SalesSetupFactory;
+use Magento\Catalog\Model\Product;
 use Magento\Catalog\Setup\CategorySetup;
 use Magento\Catalog\Setup\CategorySetupFactory;
+use Magento\Eav\Model\Entity\Attribute\ScopedAttributeInterface;
+use Magento\Eav\Model\Entity\Attribute\Source\Boolean;
+use Magento\Framework\DB\Ddl\Table;
+use Magento\Framework\Setup;
 use Magento\Sales\Model\Order;
-use Magento\Catalog\Model\Product;
-
+use Magento\Sales\Setup\SalesSetup;
+use Magento\Sales\Setup\SalesSetupFactory;
 
+/**
+ * Class InstallData
+ *
+ * @package Bazaarvoice\Connector\Setup
+ */
 class InstallData implements Setup\InstallDataInterface
```

Comment:
I wonder how we support declarative schema?  Probably not possible until the older versions of Magento are not supported anymore.

Reply:
Not sure. Agreed.